src/aero_kernel/src/utils/mod.rs

```diff
@@ -27,6 +27,22 @@ pub mod io;
 pub mod linker;
 pub mod sync;
 
+pub fn validate_ptr<T>(ptr: *const T) -> Option<&'static T> {
```

Comment:
`VirtAddr::new(ptr as usize).read_mut::<T>().as_ref()`

Reply:
fixed in https://github.com/Andy-Python-Programmer/aero/pull/65/commits/82de6408d3bf7b73374475c52d4dd5fa2ec13014